.github/actions/create-jira-ticket/action.yml

```
@@ -0,0 +1,118 @@
+            jq --arg module_name "${{ inputs.module_name }}" --arg branch "${{ github.ref_name }}" '. | map(select(.name == ($module_name + " nightly " + $branch))) | .[0].url')
+          if [ $? -ne 0 ] || [ "$run_url" == "null" ]; then
+            echo "::error::Failed to get nightly run URL"
+            exit 1
+          fi
+        fi
+
+        echo "parent_epic_id=$parent_epic_id" >> $GITHUB_OUTPUT
+        echo "parent_epic_key=$parent_epic_key" >> $GITHUB_OUTPUT
+        echo "ticket_summary=$ticket_summary" >> $GITHUB_OUTPUT
+        echo "ticket_labels=$ticket_labels" >> $GITHUB_OUTPUT
+        echo "run_url=$run_url" >> $GITHUB_OUTPUT
+      shell: bash
+      env:
+        GH_TOKEN: ${{ github.token }}
+
+    - name: Check if the ticket already exists
+      id: check_ticket
+      run: |
+        # Checking if an incident ticket already exists
+        response=$(curl \
+          --request POST \
+          --url "${{ inputs.jira_base_url }}rest/api/3/search" \
+          --user "${{ inputs.jira_user_email }}:${{ inputs.jira_api_token }}" \
+          --header "Accept:application/json" \
+          --header "Content-Type:application/json" \
+          --data '{
+              "fields": ["summary"],
+              "jql": "project = MON AND parentEpic = ${{ steps.get_context.outputs.parent_epic_key }} AND issueType = Technical AND summary ~ \"${{ steps.get_context.outputs.ticket_summary }}\" AND component = \"centreon-web\" AND resolution = unresolved ORDER BY key ASC",
+              "maxResults": 1
+            }'
+          )
+        echo "$response"
+        check_if_ticket_exists=$(echo "$response" | jq .issues[0].key)
+        if [[ "$check_if_ticket_exists" != "null" ]]; then
+          echo "abort_ticket_creation=true" >> $GITHUB_ENV
+          echo "::error::ticket found as $check_if_ticket_exists aborting ticket creation"
+        fi
+      shell: bash
+
+    - name: Create Jira Issue
+      if: ${{ env.abort_ticket_creation != 'true' }}
+      run: |
+        # Creating a new incident ticket on Jira
+        DATA=$( cat <<-EOF
+        {
+          "fields": {
+            "summary": "${{ steps.get_context.outputs.ticket_summary }}",
+            "project": {"key": "MON"},
+            "issuetype": {"id": "10209"},
+            "parent": {"id": "${{ steps.get_context.outputs.parent_epic_id }}", "key": "${{ steps.get_context.outputs.parent_epic_key }}"},
+            "labels": ${{ steps.get_context.outputs.ticket_labels }},
+            "components":[{"name": "${{ inputs.module_name }}"}],
+            "customfield_10902": {"id": "10524", "value": "DevSecOps"},
+            "customfield_10005": 1.0,
+            "description": $(cat ./.github/actions/create-jira-ticket/${{ inputs.reason_for_ticket_creation }}-ticket-template.json)
+          }
+        }
+        EOF
+        )
+
+        response=$(curl \
+          --request POST \
+          --url "${{ inputs.jira_base_url }}rest/api/3/issue" \
```

A base url should not provide trailing slash
```suggestion
          --url "${{ inputs.jira_base_url }}/rest/api/3/issue" \
```